.github/dependabot.yml

```diff
@@ -0,0 +1,6 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
```

Comment:
I guess we could also add `bundler` and `npm` here

Reply:
Well, we kind of don't need to: this repo does not keep lockfiles for those, and there's nothing for Dependabot to make PRs about for gems and JS packages.